### PR TITLE
Add runninglocally.md

### PR DESCRIPTION
**Pull request by TheRealBenjiLi**
As a new engineer, I found this pattern to be present in a lot of repos, so I thought it would be useful for new engineers to learn this before diving into code :)

### CODE REVIEW

File: runninglocally.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Running Locally
+
+There come times when you want to run your code locally on your machine. While every repo is set up differently, there are a few trends across most repos that you can fall back on. 
+
+Most repos have a `README.md`, which may have the information you're looking for. If the README does not tell you what to do, don't fret! These steps below should work for most repos:
+
+1. Check if there's a `Makefile` in the repo. If there's no Makefile, do not continue to further steps and instead ask a fellow engineer.
```
Comment (Sayan-) [inline runninglocally.md]: This pattern is mainly used in golang repos. For frontend the pattern is more along the lines:

1. `nvm use 12` (we're still migrating to node 12 IIRC, so the version will be dictated by the `Makefile`
2. `npm install`
2. `ark start -l`

**Comment by Sayan-** [comment]
Just swinging by - thanks for adding documentation :)

**Comment by alsmola** [comment]
dev-handbook should really be deprecated in my opinion - can we find a better place for this on Confluence?

**Comment by TheRealBenjiLi** [comment]
Possibly! I can try writing this information up in Confluence instead :)

**Pull request closed without merging**